.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,7 @@
 ### Changes
 
 ### Reviewers
-@baz8080 @schlan @e2po @bridgeri127 @fibelatti
+@schlan @e2po @bridgeri127 @fibelatti
```

Comment:
I should also be removed, I know Seb would also like to be taken off. I can't speak for @e2po or @fibelatti but I suspect that indeed we should all be taken off

```suggestion
```